maintainers/maintainer-list.nix

```diff
@@ -10420,6 +10420,12 @@
     githubId = 54179289;
     name = "Jason Miller";
   };
+  jmir = {
```

Comment:
Actually, this should be a separate commit. Then it will be merged soon

Reply:
Done :+1: